### ISSUE TITLE
parallax?

### ISSUE THREAD

**Issue by ghost** [studio-minus/ppg-bugs-and-requests#1679]
**Feature description**
add parallax between foreground and background so the background moves slower than the foreground

**Feature purpose**
makes the game look better and more polished

**Additional context**
No

**Comment by DindinYT37** [comment]
What makes you assume the background is that far away from the foreground?

**Comment by SpanishPizza** [comment]
because then the room would look like a giant hallway in 3D

**Comment by DindinYT37** [comment]
> because then the room would look like a giant hallway in 3D

Yeah but who says it is?

**Comment by SpanishPizza** [comment]
the wall would have to be far away for the room to be a cube, otherwise it will look like a hallway

**Comment by DindinYT37** [comment]
> the wall would have to be far away for the room to be a cube, otherwise it will look like a hallway

Yeah but like, who's saying the room isn't just like 5 meters in depth? How do you "know" that the background is apparently far away?

**Comment by SpanishPizza** [comment]
logical assumption
